intranet/apps/eighth/models.py

```diff
@@ -1248,11 +1268,13 @@ def remove_signup(self, user=None, force=False, dont_run_waitlist=False):
             return "Successfully removed signup for {}.".format(block)
 
     def accept_pass(self):
+        """Sets was_absent=False, pass_accepted=True, and saves both fields."""
```

Comment:
Probably could be reworded to become:
```suggestion
        """Accepts an eighth period pass for the EighthSignup object."""
```